examples/with-contentlayer/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "contentlayer": "^0.1.1",
+    "next": "latest",
+    "next-contentlayer": "^0.1.1",
```

Comment:
Could we update to latest here? And React 18?

Reply:
@leerob done